doc/sphinx/Doxyfile.in

```diff
@@ -917,7 +918,8 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  =  "../../src/" "../../examples/"
+INPUT                  =  "../../src/" "../../examples/" "../../doc/"
+# INPUT       = "@CMAKE_SOURCE_DIR@/src/" "@CMAKE_SOURCE_DIR@/examples/" "@CMAKE_SOURCE_DIR@/doc/"
```

Comment:
Do we need to keep this comment ?

Reply:
You can remove the comment. Will fix later. I will do some issue.

Reply:
Comment removed.